contribute/design/1.x/csi/20190606-csi-volume-provisioning.md

```diff
@@ -272,65 +272,36 @@ smooth volume create & delete operations.
 
 #### CStorVolumeConfigClass -- new custom resource
 This is a new Kubernetes custom resource that holds config information to be
-used by cstor volume claim controller while provisioning cstor volumes.
+used by cstor volume claim controller while provisioning cstor volumes. For each 
+volume corresponding CstorVolumeConfig is created.
 
 NOTE: This resource kind does not have a controller i.e. watcher logic. It holds
```

Comment:
also mention that CCC name comes as parameter through SC.. can you provide sample SC as well with parameter for CCC?